jobs/cjd_sample/README.md

```diff
@@ -0,0 +1,39 @@
+# Cloud Job Discovery client samples
+
+<a href="https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/java-docs-samples&page=editor&open_in_editor=jobs/cjd_sample/README.md">
+<img alt="Open in Cloud Shell" src ="http://gstatic.com/cloudssh/images/open-btn.png"></a>
+
+[Cloud Job Discovery][jobs] is part of Google for Jobs - a Google-wide commitment to help people find jobs more easily. Job Discovery provides plug and play access to Google’s search and machine learning capabilities, enabling the entire recruiting ecosystem - company career sites, job boards, applicant tracking systems, and staffing agencies to improve job site engagement and candidate conversion.
+
+This sample Java application demonstrates how to access the Cloud Job Discovery API using the [Google Cloud Client Library for Java][google-cloud-java].
+
+[jobs]: https://cloud.google.com/job-discovery/
+[google-cloud-java]: https://github.com/GoogleCloudPlatform/google-cloud-java
+
+## Setup
+
+1. Install [Maven](http://maven.apache.org/).
+1. [Enable](https://console.cloud.google.com/apis/api/jobs.googleapis.com/overview) Cloud Job Discovery API.
+
+## Build
+
+Build your project with:
+```
+   mvn clean package
+```
+
+## Local testing
```

Comment:
All of these samples behave a little differently in terms of how they are run. I would include at least a sentence or two on each sample to talk about what it does, how, and what the various attributes mean. 

For example, `SearchQuickstart.java` has several comments that say things like "Below job matches the job category of ADMINISTRATIVE_AND_OFFICE." But how is that when the job has no category field? Is it determined by matching terms in the job title? Is there some kind of ML that determines the category based on the description? Some more information would help to digest the contents of the samples.

Reply:
Yes, I agree. As you might detect, those code samples are written by different person for different purpose. So some of them are similar and the whole sample suites looks a little messy. I would keep modifying the samples and meanwhile you might check is there anything else we could improve. 

Thank you.